static/app/utils/theme/useChonkTheme.tsx

```diff
@@ -52,5 +53,21 @@ export function useChonkTheme(): [
     }
   }, [config.theme, organization, previousTheme, setChonkTheme]);
 
+  const chonkHotkey = useMemo(() => {
```

Comment:
wait this should be useCallback?

Reply:
er, nvm i see

Reply:
Does this actually matter?

Reply:
Somewhat yes. I opted to memo it because I wanted to pass the config to the dependency to indicate the relationship I could also `useHotkeys(ternary, [config.theme])`, but nothing prevents someone from removing config.theme as a dependency here (linters wont warn you about missing dep), which would break the functionality here

Reply:
I mean nothing prevents someone from changing this either

```
  useHotkeys(chonkHotkey, [chonkTheme.theme, chonkHotkey]);
```

```
  useHotkeys(chonkHotkey, []);
```

Reply:
Resolved in slack :)